includes/apple-exporter/class-theme.php

```diff
@@ -556,970 +556,1283 @@ private static function sort_registry( $registry ) {
 				'type'    => 'color',
 			],
-			'pullquote_border_style'             => [
+			'pullquote_border_style'               => [
 				'default' => 'solid',
 				'label'   => __( 'Pull quote border style', 'apple-news' ),
 				'options' => [ 'solid', 'dashed', 'dotted', 'none' ],
 				'type'    => 'select',
 			],
-			'pullquote_border_width'             => [
+			'pullquote_border_width'               => [
 				'default' => 3,
 				'label'   => __( 'Pull quote border width', 'apple-news' ),
 				'type'    => 'integer',
 			],
-			'pullquote_color'                    => [
+			'pullquote_color'                      => [
 				'default' => '#53585f',
 				'label'   => __( 'Pull quote color', 'apple-news' ),
 				'type'    => 'color',
 			],
-			'pullquote_color_dark'               => [
+			'pullquote_color_dark'                 => [
 				'default' => '',
 				'label'   => __( 'Pull quote color', 'apple-news' ),
 				'type'    => 'color',
 			],
-			'pullquote_font'                     => [
+			'pullquote_font'                       => [
 				'default' => 'AvenirNext-Bold',
 				'label'   => __( 'Pullquote font face', 'apple-news' ),
 				'type'    => 'font',
 			],
-			'pullquote_hanging_punctuation'      => [
+			'pullquote_hanging_punctuation'        => [
 				'default'     => 'no',
 				'description' => __( 'If set to "yes," adds smart quotes (if not already present) and sets the hanging punctuation option to true.', 'apple-news' ),
 				'label'       => __( 'Pullquote hanging punctuation', 'apple-news' ),
 				'options'     => [ 'no', 'yes' ],
 				'type'        => 'select',
 			],
-			'pullquote_line_height'              => [
+			'pullquote_line_height'                => [
 				'default' => 48.0,
 				'label'   => __( 'Pull quote line height', 'apple-news' ),
 				'type'    => 'float',
 			],
-			'pullquote_size'                     => [
+			'pullquote_size'                       => [
 				'default' => 48,
 				'label'   => __( 'Pull quote font size', 'apple-news' ),
 				'type'    => 'integer',
 			],
-			'pullquote_tracking'                 => [
+			'pullquote_tracking'                   => [
 				'default'     => 0,
 				'description' => __( '(Percentage of font size)', 'apple-news' ),
 				'label'       => __( 'Pullquote tracking', 'apple-news' ),
 				'type'        => 'integer',
 			],
-			'pullquote_transform'                => [
+			'pullquote_transform'                  => [
 				'default' => 'uppercase',
 				'label'   => __( 'Pull quote transformation', 'apple-news' ),
 				'options' => [ 'none', 'uppercase' ],
 				'type'    => 'select',
 			],
-			'screenshot_url'                     => [
+			'recipe_background_color'              => [
+				'default' => '#e1e1e1',
+				'label'   => __( 'Recipe background color', 'apple-news' ),
+				'type'    => 'color',
+			],
+			'recipe_background_color_dark'         => [
+				'default' => '',
+				'label'   => __( 'Recipe background color', 'apple-news' ),
```

Comment:
Do these dark colors need additional labeling to specify that they're for the dark variant (here and on line 1371)?